WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/EditorPage.java

```diff
@@ -106,4 +106,11 @@ public boolean publishPost() {
         waitForElementToBeDisplayed(publishConfirmation);
         return isElementDisplayed(publishConfirmation);
     }
+
+    public boolean publishPage() {
```

Comment:
This function seems to be unrelated to the activity log test as you described, if the PR wasn't intended to cover the page publish I'd recommend removing that code from this PR.